### PR TITLE
Refactor delete program

### PR DESCRIPTION
**Pull request by ArshMalik02**
Implemented delete program handler.

Comment out tests and fix them in handler/program_test.go

### CODE REVIEW

File: handler/program.go [hunk 1]
```diff
@@ -117,3 +117,85 @@ func CreateProgram(cc echo.Context) error {
...
+		}
+	}
+	if idx >= len(u.Programs) {
+		return errors.New("invalid PID")
+	}
+	toDelete := u.Programs[idx]
+	u.Programs = append(u.Programs[:idx], u.Programs[idx+1:]...)
+
+	// pref := d.Collection(programsPath).Doc(toDelete)
+	c.StoreUser(c.Request().Context(), u)
```
Comment (lumisphere902) [inline handler/program.go:155]:
```suggestion
	err = c.StoreUser(c.Request().Context(), u)
	if err != nil {
		return err
        }
```
Something quick like this should make the linter happy

File: handler/program.go [hunk 2]
```diff
@@ -117,3 +117,85 @@ func CreateProgram(cc echo.Context) error {
...
+				idx = i
+				break
+			}
+		}
+		if idx >= len(cls.Programs) {
+			return errors.New("invalid PID")
+		}
+		cls.Programs = append(cls.Programs[:idx], cls.Programs[idx+1:]...)
+
+		c.StoreClass(c.Request().Context(), cls)
```
Comment (lumisphere902) [inline handler/program.go:188]: Same here

File: handler/program.go [hunk 3]
```diff
@@ -117,3 +117,85 @@ func CreateProgram(cc echo.Context) error {
...
+			break
+		}
+	}
+	if idx >= len(u.Programs) {
+		return errors.New("invalid PID")
+	}
+	toDelete := u.Programs[idx]
+	u.Programs = append(u.Programs[:idx], u.Programs[idx+1:]...)
+
+	// pref := d.Collection(programsPath).Doc(toDelete)
```
Comment (lumisphere902) [inline handler/program.go:154]: Remove comment

**Review by lumisphere902 — CHANGES_REQUESTED** [review]
Hey Arsh, what you have here looks great! Just a couple of changes left, and one more thing: you should delete the `db/program.go` implementation, then change the handler in `server.go` to use your new version. After that, make sure to test locally, and you should be good to merge!

**Comment by ArshMalik02** [comment]
Incorporated the suggestions

**Review by lumisphere902 — APPROVED** [review]
LGTM. Thanks for all your hard work! This goes a long way into increasing the robustness of the tests and overall code quality. Feel free to merge whenever. :)

**Pull request merged**